packages/grafana-data/src/transformations/transformers/seriesToColumns.test.ts

```diff
@@ -44,38 +45,53 @@ describe('SeriesToColumns Transformer', () => {
     expect(filtered.fields).toEqual([
       {
         name: 'time',
+        state: {
+          title: 'time',
+        },
         type: FieldType.time,
         values: new ArrayVector([1000, 3000, 4000, 5000, 6000, 7000]),
         config: {},
-        labels: { origin: 'even,odd' },
+        labels: undefined,
```

Comment:
So I understand this better why is this `undefined` and not `{ name: 'even,odd' }`?

Reply:
on the join by field we do not add origin, as the "title / identity" on the column you join by (I think) should not change